judgels-backends/judgels-grader-engines/src/main/java/judgels/gabriel/languages/javascript/JavascriptNode18GradingLanguage.java

```diff
@@ -0,0 +1,37 @@
+package judgels.gabriel.languages.javascript;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import judgels.gabriel.api.GradingLanguage;
+
+public class JavascriptNode18GradingLanguage implements GradingLanguage {
```

Comment:
Nit: The language name is `JavaScript`, not `Javascript`.

Can we reflect this to all filenames and strings?